images/base/Dockerfile

```diff
@@ -74,7 +98,7 @@ RUN export ARCH=$(dpkg --print-architecture| sed 's/ppc64el/ppc64le/') \
 
 # Install crictl to /usr/local/bin
 ARG CRICTL_VERSION="v1.14.0"
-RUN export ARCH=$(dpkg --print-architecture) \
+RUN export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/') \
```

Comment:
FYI @mkumatag I think I fixed the last of these, the `ctr` binary here is also built by us and _should_ have a working PPC64LE download, but I can't test it :^)

Reply:
(the existing binary and package from canonical should also already support PPC64LE, but I took care to make sure our temporary custom one does too)